src/IO/S3/Client.cpp

```diff
@@ -798,7 +823,11 @@ void Client::slowDownAfterRetryableError() const
         UInt64 current_time_ms = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::steady_clock::now().time_since_epoch()).count();
```

Comment:
Using std::chrono::steady_clock with time_since_epoch() is unusual, but likely not a bug.